Gemfile

```diff
@@ -32,31 +33,30 @@ gem 'mas-rad_core', '0.1.3'
 gem 'oga'
 gem 'pg'
 gem 'rails_email_validator'
+gem 'rake', '~> 11'
 gem 'ransack'
 gem 'rollbar'
+gem 'rubyzip'
 gem 'sass-rails'
 gem 'sidekiq', '~> 3.3.4'
 gem 'sidekiq-unique-jobs'
 gem 'sidetiq'
 gem 'sinatra', require: false
+gem 'slack-ruby-client'
 gem 'uglifier', '>= 1.3.0'
 gem 'unicorn'
-gem 'azure-storage'
-gem 'rubyzip'
-gem 'slack-ruby-client'
-gem 'rake', '~> 11'
 
 group :test, :development do
   gem 'better_errors'
   gem 'binding_of_caller'
   gem 'factory_girl_rails'
   gem 'faker'
-  gem 'pry-rails'
+  gem 'ffaker'
   gem 'pry-byebug'
+  gem 'pry-rails'
   gem 'rspec-rails'
-  gem 'rubocop', '0.31.0'
+  gem 'rubocop', '0.49.0'
```

Comment:
Yes, @paulanthonywilson this PR fixes the vulnerability that was highlighted in the old version `0.31.0`.